rules/aip0192/trademarked_names.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0192
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/googleapis/api-linter/lint"
+	"github.com/jhump/protoreflect/desc"
+)
+
+var trademarks = map[string][]string{
+	"App Engine":     []string{"GAE", "AppEngine", "App engine"},
+	"BigQuery":       []string{"Bigquery", "Big Query", "BQ"},
+	"BigQuery ML":    []string{"BQML"},
+	"Bigtable":       []string{"BigTable", "Big Table", "Big table"},
+	"Bitbucket":      []string{"BitBucket", "Bit Bucket"},
+	"Cloud Storage":  []string{"GCS"},
+	"Compute Engine": []string{"GCE"},
+	"Dataflow":       []string{"Data Flow", "Data flow", "DataFlow"},
+	"Dataprep":       []string{"Data Prep", "Data prep", "DataPrep"},
+	"Dialogflow":     []string{"DialogFlow", "Dialog Flow", "Dialog flow"},
+	"Directory Sync": []string{"GCDS", "CDS", "DirectorySync"},
+	"GitHub":         []string{"Github", "Git Hub"},
+	"GitLab":         []string{"Gitlab", "Git Lab"},
+	"G Suite":        []string{"GSuite", "G-Suite", "gSuite"},
+	"Pub/Sub":        []string{"PubSub", "Pubsub", "Cloud Pub/Sub"},
+	"Service Mesh":   []string{"ASM", "CSM", "GCSM"},
+	"Stack Overflow": []string{"StackOverflow"},
```

Comment:
Please check for lower case also: gcs, gce, etc. Many times that is what gets into the docs.

Reply:
Done, except for `bq` (which would false positive `bq://...`).